docs/UserGuide.md

```diff
@@ -42,40 +96,27 @@ PropertyWhiz (PropertyWhiz) is a **desktop app for managing properties and prope
 
     * **`exit`** : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command, 
+6. Refer to the [Features](#features) below for details of each command, 
    or the [Command Summary](#command-summary).
 
 --------------------------------------------------------------------------------------------------------------------
-
-## Command summary
-
-Action | Format, Examples
---------|------------------
-**Add** | **Property** <br>`add property n/PROPERTY_NAME a/PROPERTY_ADDRESS s/SELLER_NAME p/SELLER_PHONE e/SELLER_EMAIL $/PRICE [t/TAG]…​` <br><br> **Buyer** <br>`add buyer n/BUYER_NAME p/BUYER_PHONE e/BUYER_EMAIL $/BUDGET) [t/TAG]…`
-**Clear** | `clear`
-**Delete** | `delete (property \| buyer) INDEX`
-**Edit** | **Property** <br>`edit property INDEX [n/PROPERTY_NAME] [a/PROPERTY_ADDRESS] [s/SELLER_NAME] [p/SELLER_PHONE] [e/SELLER_EMAIL] [$/PRICE] [([t/TAG]…​ \| [ta/TAG_TO_ADD]… [td/TAG_TO_DELETE]…)]]​`<br><br> **Buyer** <br> `edit buyer INDEX [n/BUYER_NAME] [p/BUYER_PHONE] [e/BUYER_EMAIL] [$/BUDGET]) [([t/TAG]… \| [ta/TAG_TO_ADD]… [td/TAG_TO_DELETE]…)]`
-**Find** | `find (properties \| buyers) [KEYWORDS] [t/TAG_TO_MATCH]…`
-**List** | `list`
-**Exit** | `exit`
-**Help** | `help`
-**Sort** | `sort (properties \| buyers) (price \| name) (asc \| desc)`
-**Match** | `match ( auto \| property INDEX \| buyer INDEX )`
-**Import** | `import (properties \| buyers)`
-**Export** | `export (properties \| buyers)`
-
---------------------------------------------------------------------------------------------------------------------
-
 ## Navigating the User Interface
 
 ![Ui](images/NavigatingUi.png)
 
 The UI is split into the input and output sections, as well as
 two view columns for **Property** and **Buyer** each.
 
+Item | Description
+--------|------------------
+**Input box** | Also known as the command box, this is PropertyWhiz's command line. All typed input goes here.
+**Output box** | The output box displays additional information from a command. <br> For a *successful* command, the output box may show metadata like number of entries listed. <br> For an *unsuccessful* command, such as invalid input, the output box will give detailed feedback about which part of the command is invalid and how to fix it.
+**Properties view** | A listing of properties.
+**Buyers view** | A listing of buyers.
+
 ![](images/PropertyCardUi.png)
 
-Here is the breakdown of an individual property card.
+Here is the breakdown of an individual *property card*.
 
 Item | Description
 --------|------------------
```

Comment:
Not directly relevant to Front matters, but for completion we can mention about Tags within the cards as well.